### DISCUSSION TITLE
A part of the 3D model is displayed in gray on the Android SceneViewer.

### DISCUSSION THREAD

**Discussion by tokufxug** [google/model-viewer#3630] [Q&A]
### Description

Some of the models in the GLB file are grayed out when displayed in the Android SceneViewer, but they are displayed fine on the web.
When we checked the display at the end of last year, the models were displayed correctly without being grayed out.
I have tried the latest version 1.10.1 as well as 1.9.2, 1.8 and 1.6, but no improvement.

This does not seem to be the case with all Android devices. If you have any information on this matter, or know who to contact about the problem with SceneViewer, please let us know.

![PXIiAY9Y](https://user-images.githubusercontent.com/323081/153799249-e50b22a6-a785-41e3-9ce6-db0fa21b1ed8.jpg)

#### Live Demo

https://oneplanet-org.github.io/locationar-assets/samples/tmn/20220214-001/index.html

### Version

- model-viewer: v1.10.1
- model-viewer: v1.9.2
- model-viewer: v1.8
- model-viewer: v1.6

### Device
- Galaxy S8(SCV36)
- Android 9

### Browser Affected

- [x] Chrome, version: 98.0.4758.87
- [ ] Edge
- [ ] Firefox
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [ ] Windows

### AR

- [ ] WebXR
- [x] SceneViewer
- [ ] QuickLook

**Comment by dthery** [comment]
I think you should apply your "Armature" modifier. Exporting from Blender with a modifier not applied may cause issues with the GLB. With Model-Viewer your model works well. But actually with Scene-viewer there is the problem. When I delete the modifier, I no longer have the animation, but I no longer have the problem with scene-viewer.

**Comment by julienbfabre** [comment]
We also suddenly have the same issue, with models that we were working fine before in quicklook and scene viewer, any idea what's going on? we tried the suggestion from @dthery but that solved it only on android, quicklook/ios still doesnt show the model.
